peps/pep-9999.rst

```diff
@@ -0,0 +1,292 @@
+----------------------------------------
+
+An annual term for council members is the approach taken for the Python steering council's elections. This PEP uses a cohort-based model, derived from the PSF Board's elections which enables continuity of members across a changing council.
+
+There is a trade-off between continuity of the council and reshuffles. This PEP takes the position that continuity will be more valuable for the Python Packaging space, especially combined with the vote of no confidence, automatic removal of inactive voters and regular elections.
+
+-------------------------------
+Term limits for council members
+-------------------------------
+
+While this is viewed as valuable for boards in general, this was rejected because of the size of the pool of interested and qualified people who might serve.
+
+-------------------------------
+Approval voting in the election
+-------------------------------
+
+An earlier non-public draft of this PEP utilised an approval voting process, which aligned with what :pep:`13` stated at the time of writing. Since the Python core team has consensus to move away from that and utilise BLOC-STAR, this PEP was updated to align with that for the same reasons as the core teams' move to BLOC STAR: it better captures voter intention in the results.
+
+------------------------------------------------------------------
+Disallow multiple people from the same organisation on the council
+------------------------------------------------------------------
+
+This PEP currently mirrors the Python Steering Council's limit, that at most two individuals related to a single organisation can be on the council.
+
+Limiting it to one is workable; although it hasn't come up in the SC, people do move around, and I wouldn't want good candidates to either make employment decisions based on PC membership, or have to resign based on an employment change.  Limiting it to max 2, plus votes of no confidence is probably sufficient safety from any undo employer influence.
+
+---------------------------------------------------------------------------
+Establishing specific processes for Packaging Council and PyPA relationship
+---------------------------------------------------------------------------
+
+As noted in the abstract, the focus of this PEP is on providing a minimal-but-solid foundation for further governance decisions. The specifics of this relationship would be figured out by the inaugural council.
+
+=======================================
+Appendix: Approval process for this PEP
+=======================================
+
+This PEP would likely require an atypical process for approval given that it requires changes to PyPA's governance (which involves a PyPA-committers vote) and it requires the Python Steering Council to change their delegations.
+
+To that end, the process for approval for this PEP will be:
+
+* Submit this PEP for a vote on pypa-committers, in accordance with the process outline in :pep:`609`.
```

Comment:
Is "pypa-committers" a mailing list, DPO channel, Discord channel?  Maybe be more specific?  Not all eligible voters given the above criteria may be on any particular channel.

Reply:
A mailing list.